.woodpecker.star

```diff
@@ -1920,6 +1922,7 @@ def opencloudServer(storage = "decomposed", accounts_hash_difficulty = 4, depend
         "WEB_DEBUG_ADDR": "0.0.0.0:9104",
         "WEBDAV_DEBUG_ADDR": "0.0.0.0:9119",
         "WEBFINGER_DEBUG_ADDR": "0.0.0.0:9279",
+        "STORAGE_USERS_POSIX_WATCH_FS": True,
```

Comment:
could be moved into `if storage == "posix":` because decomposefs and other storages don't need it

Reply:
deleted from .woodpecker.star 
now I set this env in the test using wrapper